### ISSUE TITLE
The MoveTank(ANALOG leftChannel... incorrectly passes arguments to the MoveTank(int8_t... method.

### ISSUE THREAD

**Issue by pbrais** [robo-crc/arduino-lib#8]
```
    void CrcLib::MoveTank(int8_t leftChannel, int8_t rightChannel, unsigned char leftMotor, unsigned char rightMotor)
    {
        SetPwmOutput(leftMotor, leftChannel);
        SetPwmOutput(rightMotor, rightChannel);
    }

    void CrcLib::MoveTank(ANALOG leftChannel, ANALOG rightChannel, unsigned char leftMotor, unsigned char rightMotor)
    {
        MoveTank(ReadAnalogChannel(leftChannel), leftMotor, ReadAnalogChannel(rightChannel), rightMotor);
                                          swap   ^   and    ^
    }
```

**Comment by sirrandalot** [comment]
Fixed in v0.1.2

**Comment by pbrais** [comment]
@sirrandalot, no, not fixed in v0.1.2.
You forgot the other MoveTank method (without the back motors, line 184).

**Comment by sirrandalot** [comment]
Fixed (hopefully finally this time) in v0.1.3
